src/main/webapp/index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world from Team 28!</h1>
```

Comment:
Along the same lines. I'd either create references to css and js from html (even when the files are empty) or not introduce files. Right now they are created for no reason and not hooked up. Feel free to merge PR as is. Let's chat about the reasons during 1-1.